PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,39 @@
+# Pull Request Template
+
+## Description
+
+Describe the changes that where made in this pull request. When possible start with a user story - short, simple descriptions of a feature told from the perspective of the person who desires the new capability. Be sure to also include the following information:
+
+
+1. Motivation for change
+2. What was changed
+3. Link to relevant issues and documentation
+
+Example:
+*As a Blockstack developer, I would like to encrypt files using the app private key. This is needed because storing unencrypted files is unacceptable. This pull request adds the* `*encryptContent*` *function which will take a string and encrypt it using the app private key. For details refer to issue #123* 
+
+
+## Type of Change
+- [ ] New feature
+- [ ] Bug fix
+- [ ] Documentation update
+- [ ] Other
+
+
+## Does this introduce a breaking change?
```

Comment:
I'd put text under this:  

> List the APIs or describe the functionality that this PR breaks.

Reply:
```Acceptance Criteria
Conditions that this pull request must satisfy to be accepted by a user, customer or other stakeholder
Additional use cases or requirements this pull request must meet to be complete```

Reply:
ignore the acceptance criteria below and move it into the description